macros/clusterAnaScurve.py

```diff
@@ -0,0 +1,234 @@
+        jobScriptName = "%s/clusterJob.sh"%dirPath
+        jobScript = open(jobScriptName, 'w+')
+        #jobScript.write('scl enable python27 - << \EOF\n')
+        jobScript.write('#!/bin/zsh\n')
+        jobScript.write('echo `python --version`\n')
+        jobScript.write('echo `gcc --version | grep "gcc"`\n')
+        jobScript.write('source /opt/rh/python27/enable\n')
+        jobScript.write('source /afs/cern.ch/sw/lcg/contrib/gcc/4.8.4/x86_64-slc6/setup.sh\n')
+        jobScript.write('export BUILD_HOME=%s\n'%buildHome)
+        jobScript.write('export DATA_PATH=%s\n'%dataPath)
+        jobScript.write('export ELOG_PATH=%s\n'%elogPath)
+        jobScript.write('source %s/cmsgemos/setup/paths.sh\n'%buildHome)
+        jobScript.write('source %s/gem-plotting-tools/setup/paths.sh\n'%buildHome)
+        jobScript.write('source $BUILD_HOME/venv/slc6/py27/bin/activate\n')
+        jobScript.write('echo `python --version`\n')
+        jobScript.write('echo `gcc --version | grep "gcc"`\n')
+
+        thisGEB = options.GEBtype
+        if chamberAndScanDatePair[0] in linkByChamber.keys():
+            thisGEB = GEBtype[linkByChamber[chamberAndScanDatePair[0]]]
+            pass
+
+        # make the python command
+        pythonCmd = 'anaUltraScurve.py -i %s -t %s --zscore=%f --ztrim=%f --maxEffPedPercent=%f --highNoiseCut=%f --deadChanCutLow=%f --deadChanCutHigh=%f'%(
+                jobInputFile,
+                thisGEB,
+                options.zscore,
+                options.ztrim,
+                options.maxEffPedPercent,
+                options.highNoiseCut,
+                options.deadChanCutLow,
+                options.deadChanCutHigh)
+        if options.calFile is not None:
+            pythonCmd += ' --calFile=%s'%(options.calFile)
+            pass
+        if options.channels:
+            pythonCmd += ' --channels'
+            pass
+        if options.extChanMapping is not None:
+            pythonCmd += ' --extChanMapping=%s'%(options.extChanMapping)
+            pass
+        if options.performFit:
+            pythonCmd += ' --fit'
+            pass
+        if options.PanPin:
+            pythonCmd += ' --panasonic'
+            pass
+        pythonCmd += '\n'
+        
+        jobScript.write(pythonCmd)
+        #jobScript.write('EOF')
+        jobScript.close()
+        runCommand( ['chmod', '+x', jobScriptName] )
+
+        jobCmd = [
+                'bsub',
+                '-q',
+                options.queue,
+                '-o',
+                "%s/jobOut.txt"%jobStdOut,
+                '-e',
+                "%s/jobErr.txt"%jobStdErr,
+                jobScriptName ]
+
+        if options.debug:
+            print(idx, jobCmd)
+            pass
+        else:
+            runCommand(jobCmd)
+            time.sleep(1)
+            pass
+        pass # end loop over listChamberAndScanDate
+
+    print("Job submission completed")
+    print("To check the status of your jobs execute:")
+    print("")
```

Comment:
Is there any particular reason to use `print("")` instead of adding \n in preceding print statement?

Reply:
I find readability of the code enhanced in this way as I often miss special characters such as `\n`.

Reply:
you can also use triple quotes for multi-line strings